connect/cli/plugins/product/sync/actions.py

```diff
@@ -67,23 +68,38 @@ def sync(self):  # noqa: CCR001
                     action = self._client.products[self._product_id].actions[
                         data.verbose_id
                     ].update(payload)
+                    self._update_sheet_row(ws, row_idx, action)
                     self._mstats.updated()
                 except Exception as e:
                     self._mstats.error(str(e), row_idx)
 
             if data.action == 'create':
                 try:
+                    original_action = list(filter(lambda x: x['action'] == data.id, actions))
+                    if original_action:
+                        self._updated_or_skipped(ws, row_idx, original_action, payload)
```

Comment:
I think it makes more sense to pass `original_action[0]`, so the function does not expect a list with just one item

Reply:
Yes!, you are totally right, there's no sense to pass a list with only one item, I will update this.